subsys/debug/tracing/cpu_stats.c

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```

Comment:
Zephyr follows Apache 2 not BSD with extras.

Reply:
fixed